README.md

```diff
@@ -215,3 +215,38 @@ The dashboard shows that the load is dispatched among the baristas.
     ```bash
     ./delete-kubernetes.sh
     ```
+
+# Helm (v3)
+
+Note: current problems: Although Strimzi's kafkatopics resource creates the 'orders' topic with multiple partitions, the partition count then seems to get reset to 1, which means Keda can't scale the barista. As a workaround, changing the partition count and then upgrading the chart seems to resolve this (usually).
```

Comment:
I assume the partition count is changed in the file? If so, there is a script you can run locally as well, seeing as you have nodeport on kafka:
`./kafka-topics.bat --bootstrap-server localhost:32100 --alter --topic orders--partitions 5`

This way you don't need to upgrade the chart after. I'm not sure how useful this is though, so up to you!

Reply:
Oh good point, though I can probably get rid of this statement.  I think I have resolved it by creating the Kafka cluster as an earlier step.  If Kafka and Strimzi are already running, the topics seem to be created with the right parameters reliably.